packages/angular/src/lib/utils/decorators.ts

```diff
@@ -4,11 +4,17 @@ export interface ZonedClass {
   ngZone: NgZone;
 }
 
-export function OutsideZone<T extends ZonedClass>(targetClass: T, functionName: string, descriptor) {
+export function OutsideZone<T extends ZonedClass>(
```

Comment:
Is it user anywhere?
Class decorators breaks tree shake. Prefer to remove it completely

Reply:
it's a functions decorator, being used in the Angular implementation of the diagram itself, wraps mouse handling functions